tests/bgp/test_bgp_queue.py

```diff
@@ -59,6 +59,10 @@ def test_bgp_queues(duthosts, enum_frontend_dut_hostname, enum_asic_index, tbinf
     for k, v in list(bgp_facts['bgp_neighbors'].items()):
         # Only consider established bgp sessions
         if v['state'] == 'established':
+            # For "peer group" if it's internal it will be "INTERNAL_PEER_V4" or "INTERNAL_PEER_V6"
```

Comment:
Could you also leave the sample value when it's external BGP peers?

Reply:
Sure I've added one for external